apps/etterlatte-tidshendelser/src/main/kotlin/no/nav/etterlatte/tidshendelser/OpprettJobberJobb.kt

```diff
@@ -64,6 +64,6 @@ class OpprettJobb(
         OMS_DOED_4MND(JobbType.OMS_DOED_4MND, 1, 0),
         OMS_DOED_6MND(JobbType.OMS_DOED_6MND, 1, 0),
         OMS_DOED_6MND_INFORMASJON_VARIG_UNNTAK(JobbType.OMS_DOED_6MND_INFORMASJON_VARIG_UNNTAK, 8, 0),
-        // TODO: EY-4519 legg inn når ferdigtestet OMS_DOED_12MND(JobbType.OMS_DOED_12MND, 1, 0),
+        OMS_DOED_12MND(JobbType.OMS_DOED_12MND, 1, 0),
```

Comment:
```suggestion
        OMS_DOED_12MND(JobbType.OMS_DOED_12MND, 1, -2),
```

Jeg tror denne må være offsettet med -2 måneder for å treffe riktige saker -- det var kjøringen for behandlingsmaaned 2024-10 som identifiserte saker med avdøds dødsdato i dec 2023, og hvis den kjører i desember første gang vil vi måtte treffe -2 måneder fra dec.

Men vi kan evt. bare hoppe over å opprette kjøringen nå med en gang, og så tar vi det på mandag